src/cases/containers/allocate-role-container/allocate-role-container.component.spec.ts

```diff
@@ -0,0 +1,73 @@
+import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { ActivatedRoute } from '@angular/router';
+import { RouterTestingModule } from '@angular/router/testing';
+import { Store } from '@ngrx/store';
+import { MockStore, provideMockStore } from '@ngrx/store/testing';
+import { of } from 'rxjs/internal/observable/of';
+
+import { State } from '../../../app/store';
+import { RoleAllocationType } from '../../../cases/enums';
+import { RoleAllocationConstants } from '../../components/constants';
+
+import { AllocateRoleContainerComponent } from '..';
+
+describe('AllocateRoleContainerComponent', () => {
+  let component: AllocateRoleContainerComponent;
```

Comment:
would this best to move to src\work-allocation-2\**containers**, if this is only used in work allocation 2.